PayItForward.Data.Services/UsersService.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace PayItForward.Services.Data
+{
+    using System;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using PayItForward.Data;
+    using PayItForward.Services.Data.Abstraction;
+    using Dbmodel = PayItForward.Data.Models;
+
+    // TO DO :
+    // Bisness logic implementation later when Constructors are created
+
+    public class UsersService : IUsersService
+    {
+        private readonly UsersRepository<Dbmodel.User, string> usersRepo;
+
+        public UsersService()
+        {
+            this.usersRepo = new UsersRepository<Dbmodel.User, string>(new PayItForwardDbContext());
+        }
+
```

Comment:
You can add more business logic here, not only CRUD operations. Example: Get All users from particular role